src/sage/rings/number_field/S_unit_solver.py

```diff
@@ -1,10 +1,10 @@
 # sage.doctest: needs sage.rings.number_field sage.rings.padics
 r"""
-Solve `S`-unit equation `x + y = 1`
+Solve for the `S`-unit equation `x + y = 1`
```

Comment:
What I meant is that this section title is a verb phrase, whereas the others are noun phrases

Reply:
Ah, sorry. Right. Fixed.